app/extensions/brave/locales/en-US/extensions.properties

```diff
@@ -29,3 +29,5 @@ torrent= Torrent Viewer
 torrentDesc=Uses WebTorrent to display torrents directly in the browser. Supports torrent files and magnet links.
 vimium= Vimium
 vimiumDesc=
+honey=Honey
```

Comment:
I don't think this is needed, but it doesn't hurt anything... Since the extension is a product name, it shouldn't be translated. But... using Japanese as an example... I could see writing this word using Katakana more helpful to the user. cc: @luixxiul